apps/taskmasters/flowlord/taskmaster.go

```diff
@@ -54,12 +59,43 @@ func (tm *taskMaster) Info() interface{} {
 	}
 }
 
+func (tm *taskMaster) CacheUpdate(ctx context.Context) {
```

Comment:
CacheUpdate implies this is running once to update the cache. AutoUpdate is more descriptive or what it is doing. It would also be good to add a comment above the method to describe what it is doing.